source/renderer/app/components/wallet/receive/Address.js

```diff
@@ -44,26 +29,19 @@ export default class Address extends Component<Props> {
       shareAddressLabel,
       copyAddressLabel,
       isIncentivizedTestnet,
-      isShowingSubMenus,
-      minCharsInit,
-      minCharsEnd,
+      // isShowingSubMenus,
```

Comment:
@daniloprates Why not removing these lines instead of comments?
We can restore them if we need them later.

Reply:
> @daniloprates Great work. 👍 Just minor comments left.

Hey @yakovkaravelov this PR is still in WIP and this solution doesn't work for monospaced fonts, unfortunately.